.eslintrc

```diff
@@ -14,5 +14,6 @@
     "no-underscore-dangle": [0],
     "no-unused-vars": [0],
     "semi": [0],
+    "consistent-return": [0],
```

Comment:
good rule to turn off. i'm not sure if we lose anything by turning it off, but we should be all right. I think we just need to remember to get rid of returns that dont matter.